ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -851,6 +855,7 @@ gru_cell_activation_function
 onnx_model_quant_conv_linear
 onnx_model_quant_conv_linear_2d
 onnx_model_quant_conv_linear_3d
+IE_GPU.round_int64
```

Comment:
Please move this test to the GPU section after line 1134

Reply:
Done